CHANGELOG.md

```diff
@@ -2,9 +2,22 @@
 
 All notable changes to this project will be documented in this file.
 
+## v2.4.0
+
+### Features and Enhancements:
+
+- Support for Trace Analytics [#122](https://github.com/grafana/opensearch-datasource/pull/122), @idastambuk, @katebrenner, @iwysiu, @sarahzinger
+
+### Bug Fixes:
+
+- Update Backend Dependencies [#148](https://github.com/grafana/opensearch-datasource/pull/148), @fridgepoet
+- Cspell bug fix [#146](https://github.com/grafana/opensearch-datasource/pull/146), @iwysiu
```

Comment:
In my opinion, adding a username to the spellcheck omission doesn't need to be part of the changelog as we should keep this fairly concise. Not sure about the backend dependencies above either.